### ISSUE TITLE
-- Logging error in Loguru Handler #1 ---

### ISSUE THREAD

**Issue by tigerdhl** [Megvii-BaseDetection/YOLOX#1104]
I get the error at the start of training the default yolox-s model, but it is no problem in the last version yolox
-----------------------------------------------------------
dell@dell-Precision-7820-Tower:~/DLproject/YOLOX$ python3 tools/train.py -f exps/example/yolox_voc/yolox_voc_s.py -d 0 -b 32 -c weights/yolox_s.pth
2022-01-26 14:18:48 | INFO     | yolox.core.trainer:126 - args: Namespace(batch_size=32, cache=False, ckpt='weights/yolox_s.pth', devices=0, dist_backend='nccl', dist_url=None, exp_file='exps/example/yolox_voc/yolox_voc_s.py', experiment_name='yolox_voc_s', fp16=False, machine_rank=0, name=None, num_machines=1, occupy=False, opts=[], resume=False, start_epoch=None)
2022-01-26 14:18:48 | INFO     | yolox.core.trainer:127 - exp value:
╒══════════════════╤════════════════════════════╕
│ keys             │ values                     │
╞══════════════════╪════════════════════════════╡
│ seed             │ None                       │
├──────────────────┼────────────────────────────┤
│ output_dir       │ './YOLOX_outputs'          │
├──────────────────┼────────────────────────────┤
│ print_interval   │ 10                         │
├──────────────────┼────────────────────────────┤
│ eval_interval    │ 10                         │
├──────────────────┼────────────────────────────┤
│ num_classes      │ 9                          │
├──────────────────┼────────────────────────────┤
│ depth            │ 0.33                       │
├──────────────────┼────────────────────────────┤
│ width            │ 0.5                        │
├──────────────────┼────────────────────────────┤
│ act              │ 'silu'                     │
├──────────────────┼────────────────────────────┤
│ data_num_workers │ 4                          │
├──────────────────┼────────────────────────────┤
│ input_size       │ (640, 640)                 │
├──────────────────┼────────────────────────────┤
│ multiscale_range │ 5                          │
├──────────────────┼────────────────────────────┤
│ random_size      │ (14, 26)                   │
├──────────────────┼────────────────────────────┤
│ data_dir         │ None                       │
├──────────────────┼────────────────────────────┤
│ train_ann        │ 'instances_train2017.json' │
├──────────────────┼────────────────────────────┤
│ val_ann          │ 'instances_val2017.json'   │
├──────────────────┼────────────────────────────┤
│ test_ann         │ 'instances_test2017.json'  │
├──────────────────┼────────────────────────────┤
│ mosaic_prob      │ 1.0                        │
├──────────────────┼────────────────────────────┤
│ mixup_prob       │ 1.0                        │
├──────────────────┼────────────────────────────┤
│ hsv_prob         │ 1.0                        │
├──────────────────┼────────────────────────────┤
│ flip_prob        │ 0.5                        │
├──────────────────┼────────────────────────────┤
│ degrees          │ 10.0                       │
├──────────────────┼────────────────────────────┤
│ translate        │ 0.1                        │
├──────────────────┼────────────────────────────┤
│ mosaic_scale     │ (0.1, 2)                   │
├──────────────────┼────────────────────────────┤
│ mixup_scale      │ (0.5, 1.5)                 │
├──────────────────┼────────────────────────────┤
│ shear            │ 2.0                        │
├──────────────────┼────────────────────────────┤
│ enable_mixup     │ True                       │
├──────────────────┼────────────────────────────┤
│ warmup_epochs    │ 1                          │
├──────────────────┼────────────────────────────┤
│ max_epoch        │ 300                        │
├──────────────────┼────────────────────────────┤
│ warmup_lr        │ 0                          │
├──────────────────┼────────────────────────────┤
│ basic_lr_per_img │ 0.00015625                 │
├──────────────────┼────────────────────────────┤
│ scheduler        │ 'yoloxwarmcos'             │
├──────────────────┼────────────────────────────┤
│ no_aug_epochs    │ 15                         │
├──────────────────┼────────────────────────────┤
│ min_lr_ratio     │ 0.05                       │
├──────────────────┼────────────────────────────┤
│ ema              │ True                       │
├──────────────────┼────────────────────────────┤
│ weight_decay     │ 0.0005                     │
├──────────────────┼────────────────────────────┤
│ momentum         │ 0.9                        │
├──────────────────┼────────────────────────────┤
│ exp_name         │ 'yolox_voc_s'              │
├──────────────────┼────────────────────────────┤
│ test_size        │ (640, 640)                 │
├──────────────────┼────────────────────────────┤
│ test_conf        │ 0.01                       │
├──────────────────┼────────────────────────────┤
│ nmsthre          │ 0.65                       │
╘══════════════════╧════════════════════════════╛
/home/dell/anaconda3/envs/tcstorch/lib/python3.8/site-packages/torch/functional.py:445: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  /opt/conda/conda-bld/pytorch_1639180588308/work/aten/src/ATen/native/TensorShape.cpp:2157.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
2022-01-26 14:18:49 | INFO     | yolox.core.trainer:132 - Model Summary: Params: 8.94M, Gflops: 26.65
2022-01-26 14:18:51 | INFO     | yolox.core.trainer:286 - loading checkpoint for fine tuning
2022-01-26 14:18:51 | WARNING  | yolox.utils.checkpoint:24 - Shape of head.cls_preds.0.weight in checkpoint is torch.Size([80, 128, 1, 1]), while shape of head.cls_preds.0.weight in model is torch.Size([9, 128, 1, 1]).
2022-01-26 14:18:51 | WARNING  | yolox.utils.checkpoint:24 - Shape of head.cls_preds.0.bias in checkpoint is torch.Size([80]), while shape of head.cls_preds.0.bias in model is torch.Size([9]).
2022-01-26 14:18:51 | WARNING  | yolox.utils.checkpoint:24 - Shape of head.cls_preds.1.weight in checkpoint is torch.Size([80, 128, 1, 1]), while shape of head.cls_preds.1.weight in model is torch.Size([9, 128, 1, 1]).
2022-01-26 14:18:51 | WARNING  | yolox.utils.checkpoint:24 - Shape of head.cls_preds.1.bias in checkpoint is torch.Size([80]), while shape of head.cls_preds.1.bias in model is torch.Size([9]).
2022-01-26 14:18:51 | WARNING  | yolox.utils.checkpoint:24 - Shape of head.cls_preds.2.weight in checkpoint is torch.Size([80, 128, 1, 1]), while shape of head.cls_preds.2.weight in model is torch.Size([9, 128, 1, 1]).
2022-01-26 14:18:51 | WARNING  | yolox.utils.checkpoint:24 - Shape of head.cls_preds.2.bias in checkpoint is torch.Size([80]), while shape of head.cls_preds.2.bias in model is torch.Size([9]).
--- Logging error in Loguru Handler #1 ---
Record was: {'elapsed': datetime.timedelta(seconds=4, microseconds=846932), 'exception': (type=<class 'xml.etree.ElementTree.ParseError'>, value=ParseError('no element found: line 1, column 0'), traceback=<traceback object at 0x7fc68ebe9180>), 'extra': {}, 'file': (name='launch.py', path='/home/dell/DLproject/YOLOX/yolox/core/launch.py'), 'function': 'launch', 'level': (name='ERROR', no=40, icon='❌'), 'line': 98, 'message': "An error has been caught in function 'launch', process 'MainProcess' (13798), thread 'MainThread' (140495071781696):", 'module': 'launch', 'name': 'yolox.core.launch', 'process': (id=13798, name='MainProcess'), 'thread': (id=140495071781696, name='MainThread'), 'time': datetime(2022, 1, 26, 14, 18, 52, 870773, tzinfo=datetime.timezone(datetime.timedelta(seconds=28800), 'CST'))}

**Comment by FateScript** [comment]
Your ckpt mismatched.
Look at the following log:
```
2022-01-26 14:18:51 | WARNING | yolox.utils.checkpoint:24 - Shape of head.cls_preds.2.bias in checkpoint is torch.Size([80]), while shape of head.cls_preds.2.bias in model is torch.Size([9]).
```

**Comment by tigerdhl** [comment]
> Your ckpt mismatched. Look at the following log:
> 
> ```
> 2022-01-26 14:18:51 | WARNING | yolox.utils.checkpoint:24 - Shape of head.cls_preds.2.bias in checkpoint is torch.Size([80]), while shape of head.cls_preds.2.bias in model is torch.Size([9]).
> ```

an xml file is miss context,thanks
